spec/controllers/users_controller_spec.rb

```diff
@@ -106,6 +107,31 @@ def valid_user_attributes
             )
           end
         end
+
+        context "when the previous user's account was purged" do
```

Comment:
Could you move this test to `admin_tasks.rake_spec`?

The official reason is to match the file of the code under test.
The real reason is this will introduce conflicts with #3378.

Reply:
I can test for the properties of the invitation there, but checking the controller's response there would require a lot of code duplication (and would introduce compatibility issues with #3378 anyway, since I believe it changes both the parameter name and the path).

Are you okay with not checking the controller?

Reply:
Could we punt the tests to another issue and add them sometime after we've finished Devise? We can test manually for now. 

(I foolishly didn't think about potential conflicts yesterday morning before approving this issue for a quick single-issue deploy while we updated the Devise branch /o\)

Reply:
I reverted the test changes in `spec/controllers/users_controller_spec.rb` and added a couple of tests for the purge account task (including one that checks the fields on the invitation). It doesn't check the controller's response, but it should be compatible with #3378 without requiring any extra code changes.